airbyte-integrations/connectors/destination-mysql/gradle.properties

```diff
@@ -0,0 +1,3 @@
+# our testcontainer has issues with too much concurrency.
+# 4 threads seems to be the sweet spot.
+testExecutionConcurrency=4
```

Comment:
can we file a github issue to be able to set a max concurrency at the container level, so that we can run as many tests as possible in parallel without running into some weird stuff at the DB instance level? Or investigate why the container starts behaving weirdly past a certain concurrency level. Because 4 concurrent transaction should really be nothing for any RBDMS

Reply:
integration tests are just hard timing out now >.> submitted https://github.com/airbytehq/airbyte-internal-issues/issues/7240